backend/src/main/java/com/summerHack/diningTogether/utils/MapUtils.java

```diff
@@ -0,0 +1,13 @@
+package com.summerHack.diningTogether.utils;
+
+import okhttp3.*;
+
+
+import java.io.IOException;
+
+public interface MapUtils {
```

Comment:
Usually `utils` are only for class with static methods. If you want to use dependency injection, you should name it as `service` instead.